lang_python.adoc

```diff
@@ -17,21 +17,29 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 kaitai-struct-compiler -t python my_spec.ksy
 
 This normally generates `my_spec.py` parser module, which is compatible
-with both Python 2 and 3.
+with both Python 2 and 3. It is strongly recommended to use python 3
+because python 2 is EOL since 2020-01-01.
 
-TODO: document `--python-package`
+`--python-package` allows you to select a parent package name used by
+one generated file to import symbols from another one. If you want to
+embed KSC-generated files into an own package, it is recommended to
+set it to `.`, which means "current package" rather than to the real name
+of the subpackage, because setting it to a real name will cause breakage
+when you want to reorganize or rename your package. 
 
 [[add-runtime]]
 === Adding runtime library
 
 Code generated by Kaitai Struct compiler will require additional runtime
-library (https://pypi.org/project/kaitaistruct/[kaitaistruct at PyPi],
+library (https://pypi.org/project/kaitaistruct/[image:https://img.shields.io/pypi/v/kaitaistruct.svg[kaitaistruct at PyPi]],
 https://github.com/kaitai-io/kaitai_struct_python_runtime[sources at
 GitHub]).
 
 Runtime library API version must match compiler version that you're using
 (i.e. if you use ksc v0.8, then you must use compatible v0.8 runtime).
 
+If you are using non-release KSC, you should also use non-release runtime.
```

Comment:
I don't like the term non-release here, IMHO something like "latest unstable" would do better.

Reply:
thanks, fixed. Found a neutral term `snapshot`.